lib/src/builders/method.dart

```diff
@@ -176,6 +212,7 @@ abstract class MethodBuilder
   /// Creates a new [MethodBuilder].
   factory MethodBuilder(
     String name, {
+    MethodModifier modifier,
```

Comment:
Can a method have multiple modifiers? I'm not terribly versed in the Dart terminology, but in Java `private`, `protected`, `static`, `final`, etc are [all considered "modifiers"](https://www.tutorialspoint.com/java/java_modifier_types.htm) and can be combined.